pkgs/games/prismlauncher/default.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  jdk8,
+  jdk,
+  zlib,
+  file,
+  wrapQtAppsHook,
+  xorg,
+  libpulseaudio,
+  qtbase,
+  libGL,
+  quazip,
+  glfw,
+  openal,
+  extra-cmake-modules,
+  tomlplusplus,
+  ghc_filesystem,
+  msaClientID ? "",
+  jdks ? [jdk jdk8],
+}: let
+  libnbtplusplus = fetchFromGitHub {
+    owner = "ljfa-ag";
+    repo = "libnbtplusplus";
+    rev = "v2.5";
+    sha256 = "sha256-4PHKPbWIWAy9ym25M4SnllTkrqnR+g/0AOqyMBZcdX4=";
+  };
```

Comment:
Upstream version of libnbtplusplus breaks link-time-optimizations in Prism Launcher.
Apart from that there have been a few modifications ever since it was originally forked by the MultiMC project. That's why I would propose to keep using our fork.

See https://github.com/PrismLauncher/libnbtplusplus/compare/c7cb857c2fe0a066011cf7d2a8f82eb025d86fec...PrismLauncher:libnbtplusplus:master